.evergreen/tasks.yml

```diff
@@ -25,7 +25,7 @@ tasks:
       - func: bootstrap
       - func: test-electron
         vars:
-          debug: 'hadron*,mongo*'
+          debug: 'hadron*,mongo*,*COMPASS-CONNECT-UI*'
```

Comment:
Trying to see what's going on here https://github.com/mongodb-js/compass/blob/c18ea4d531f14aea20b161ec1c563ea340eed80f/packages/compass-connections/src/modules/telemetry.ts#L51